src/core/modules/Landscape.hpp

```diff
@@ -310,10 +310,19 @@ class LandscapeOldStyle : public Landscape
 	} landscapeTexCoord;
 
 private:
-	void drawFog(StelCore* core, StelPainter&) const;
+	// Low-GL versions draw in the bad old way, suitable for low OpenGL versions like GL<3 or GLES2.
+	// This makes polygons stick in all directions on large FoV in some projections.
```

Comment:
They are not just missing, they are also insufficiently tesselated. Thus, the directions where they point to are wrong because of the lack of required curvature.

Reply:
Ah, OK. 

I think the idea of libtess (which has fixed-function OpenGL1 pipeline in mind) would be to add callbacks for triangles with "dangerous" normals so that these could be further split. Currently it seems to me these triangles are just skipped, leaving a ripped edge.  But TBH this library is nothing I want to debug :-o

This was also the reason why I never allowed semitransparent polygonal landscapes despite their usefulness. These showed overlapping edge polygons. No problem if solid, but ugly with transparency. Maybe a shader-based solution can alleviate this. Should we add a "transparency" option to the landscapes dialog?

Reply:
Well, this is a job for another PR. I suppose shader-based solution may be capable of transparency.

Reply:
Yes, sure, another PR.